core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -118,37 +119,46 @@ var ExplorationEditorFeedbackTab = function() {
     var matchingRow = element(by.cssContainingText(
       `${suggestionRowClassName} ${feedbackSubjectClassName}`,
       suggestionDescription));
-    expect(await matchingRow.isDisplayed()).toBe(true);
-    await matchingRow.click();
-    expect(await viewSuggestionButton.isDisplayed()).toBe(true);
-    await viewSuggestionButton.click();
-    expect(await rejectSuggestionButton.isDisplayed()).toBe(true);
-    await suggestionReviewMessageInput.sendKeys('Review message');
-    await rejectSuggestionButton.click();
+    await action.click(
+      'Matching Suggestion Row and feedback subject', matchingRow);
+    await action.click('View Suggestion Button', viewSuggestionButton);
+    await action.sendKeys(
+      'Suggestion Review Message Input',
+      suggestionReviewMessageInput, 'Review message');
+    await action.click('Reject Suggestion Button', rejectSuggestionButton);
     await waitFor.invisibilityOf(
       acceptSuggestionButton, 'Suggestion modal takes too long to disappear');
     await waitFor.pageToFullyLoad();
   };
 
   this.selectLatestFeedbackThread = async function() {
-    await waitFor.visibilityOf(
-      await element.all(by.css(suggestionRowClassName)).first(),
-      'No feedback messages are visible.');
-    await element.all(by.css(suggestionRowClassName)).first().click();
+    var suggestionRowFirst =
+      element.all(by.css(suggestionRowClassName)).first();
```

Comment:
Can you wrap this inside `(..)` and break after `(` (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript))?

```suggestion
    var suggestionRowFirst = (
      element.all(by.css(suggestionRowClassName)).first());
```